packages/teams-js/test/private/videoEx.spec.ts

```diff
@@ -249,6 +243,71 @@ describe('videoEx', () => {
         } as DOMMessageEvent);
         expect(handlerInvoked).toBe(false);
       });
+
+      describe('mediaStream', () => {
+        beforeEach(async () => {
+          await utils.initializeWithContext(FrameContexts.sidePanel);
+          utils.setRuntimeConfig({ apiVersion: 1, supports: { video: { mediaStream: true } } });
+        });
+
+        it('should get and register stream with streamId received from startVideoExtensibilityVideoStream', async () => {
+          expect.assertions(4);
+
+          // Arrange
+          const videoFrameHandler = jest.fn();
+          const webview = window['chrome']['webview'] as unknown as {
+            getTextureStream: jest.Mock;
+            registerTextureStream: jest.Mock;
+          };
+
+          // Act
+          videoEx.registerForVideoFrame({
+            ...registerForVideoFrameParameters,
+            videoFrameHandler,
+          });
+          utils.respondToFramelessMessage({
+            data: {
+              func: 'video.startVideoExtensibilityVideoStream',
+              args: [{ streamId: 'stream id' }],
+            },
+          } as DOMMessageEvent);
+          await utils.flushPromises();
+
+          // Assert
+          expect(webview.getTextureStream).toHaveBeenCalledTimes(1);
```

Comment:
I don't see any unit tests that are verifying that the `video.mediaStream.audioInferenceDiscardStatusChange` handler is getting registered for successfully

Reply:
added